core/modules/layout/includes/layout.layout.inc

```diff
@@ -76,6 +76,21 @@ function layout_layout_context_info() {
   return $info;
 }
 
+/**
+ * Implements hook_layout_relationship_info().
+ */
+function layout_layout_relationship_info() {
+  $info['author_from_node'] = array(
+    'title' => t('Author from content'),
+    'class' => 'LayoutRelationshipAuthorFromNode',
+    'context' => 'node',
+    'context_label' => t('Node'),
+    'description' => t('Creates the author of a node as a user context.'),
```

Comment:
`Creates the author of a node as a...` should probably also be `Creates the author of content as a...` or `Creates the author of a piece of content as a...`